### PR TITLE
Readme updates for formatting and samples

### PR DESCRIPTION
**Pull request by Eilon**
- Changed HTTP URL to HTTPS
- Fixed list formatting
- Added sample to show creation of GraphQLHttpClient

### CODE REVIEW

**Comment by Eilon** [comment]
I figured these changes are fairly trivial, so no point logging an issue. But I'm happy to follow any process that is required.

File: README.md [hunk 1]
```diff
@@ -6,11 +6,19 @@ A GraphQL Client for .NET Standard over HTTP.
 
 ## Specification:
 The Library will try to follow the following standards and documents:
-[GraphQL Specification](https://facebook.github.io/graphql/June2018)
-[GraphQL HomePage](http://graphql.org/learn)
+
+* [GraphQL Specification](https://facebook.github.io/graphql/June2018)
```
Comment (sungam3r) [inline README.md:10]: This link redirects to http://spec.graphql.org/ for a long time

```suggestion
* [GraphQL Specification](https://spec.graphql.org/June2018/)
```

Comment (Eilon) [reply README.md]: Good suggestion; accepted!

**Comment by Eilon** [comment]
Thank you!

**Pull request merged**